regression_tests/stages/build_platform_helper.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+cd "${CODEBUILD_SRC_DIR}"
```

Comment:
We decided to explicitly change into the correct directory on all stages to avoid potential mixups.